File: app/src/main/java/com/lamnt/foodorder/view/fragment/auth/RegisterFragment.java
package com.lamnt.foodorder.view.fragment.auth;

import android.view.View;

import com.lamnt.foodorder.R;
import com.lamnt.foodorder.view.fragment.base.BaseFragment;

public class RegisterFragment extends BaseFragment {
    @Override
    public int getLayoutRes() {
        return R.layout.fragment_register;
    }

    @Override
    protected void unit(View v) {

    }

    @Override
    protected boolean isShowNotificationIcon() {
        return false;
    }

    @Override
    protected boolean isShowSearchIcon() {
        return false;
    }

    @Override
    protected boolean isShowBottomNav() {
        return false;
    }
}

File: app/src/main/java/com/lamnt/foodorder/view/fragment/auth/RegisterFragment.kt
package com.lamnt.foodorder.view.fragment.auth

import android.view.View
import com.lamnt.foodorder.R
import com.lamnt.foodorder.view.fragment.base.BaseFragment

class RegisterFragment : BaseFragment() {
    override fun getLayoutRes(): Int {
        return R.layout.fragment_register
    }

    override fun setTitle(): Int {
        return 0
    }

    override fun setViewOnClick(): List<View> {
        return listOf()
    }

    override fun onViewClicked(id: Int) {
        TODO("Not yet implemented")
    }

    override fun unit() {}

    override val isShowNotificationIcon: Boolean
        get() = false

    override val isShowSearchIcon: Boolean
        get() = false

    override val isShowBottomNav: Boolean
        get() = false
}